src/bun.js/bindings/JSS3Bucket.cpp

```diff
@@ -0,0 +1,213 @@
+
+#include "root.h"
+
+#include "JavaScriptCore/JSType.h"
+#include "JavaScriptCore/JSObject.h"
+#include "JavaScriptCore/JSGlobalObject.h"
+#include <JavaScriptCore/InternalFunction.h>
+#include "ZigGeneratedClasses.h"
+
+#include "JSS3Bucket.h"
+#include <JavaScriptCore/ObjectConstructor.h>
+#include <JavaScriptCore/FunctionPrototype.h>
+#include "JSS3File.h"
+#include "JavaScriptCore/JSCJSValue.h"
+#include "ErrorCode.h"
+
+namespace Bun {
+using namespace JSC;
+
+// External C functions declarations
+extern "C" {
+SYSV_ABI void* JSS3Bucket__construct(JSC::JSGlobalObject*, JSC::CallFrame* callframe);
```

Comment:
SYSV_ABI is this basically `__attribute__((sysv_abi))` right?

Reply:
Yeah, which is relevant for windows

Reply:
I was reading about it good catch will keep it in mind when doing extern "C" next time